src/style.css

```
@@ -1 +1,17 @@
 /* styles go here */
+fieldset {
+  margin-bottom: 20px;
+}
+
+label {
+  display: block;
+  margin-bottom: 10px;
+}
+
+div label {
```

```suggestion
label {
  display: block;
  margin-bottom: 10px;
}

div label {
```
Краще створити відповідні класи і стилізувати по класам, а не по тегам